app/uk/gov/hmrc/securemessage/controllers/SecureMessageController.scala

```diff
@@ -145,38 +145,55 @@ class SecureMessageController @Inject()(
         }
   }
 
-  def getContentDetail(rawId: String): Action[AnyContent] = Action.async { implicit request =>
-    decodePath(rawId) match {
+  def getMessages(
+    enrolmentKeys: Option[List[String]],
+    customerEnrolments: Option[List[CustomerEnrolment]],
+    tags: Option[List[FilterTag]]): Action[AnyContent] =
+    Action.async { implicit request =>
+      {
+        validateQueryParameters(request.queryString, "enrolment", "enrolmentKey", "tag") match {
```

Comment:
I thought we got rid of enrolment?

Reply:
according to Richard, you removed it, following a discussion with Dominic.

Reply:
no, i just reverted to what was before so no acceptance test were changed. This is not part of this ticket anyway :)

Reply:
ok, my mistake, I thought we reverted your original PR a couple of weeks ago because enrolments were removed.